test_dub_package_impl.sh

```diff
@@ -0,0 +1,18 @@
+test_dub_package_impl() {
```

Comment:
`set -euxo pipefail`

Reply:
This is file is intended to be sourced in other files.

Reply:
Not relevant anymore.